package.json

```diff
@@ -137,11 +142,14 @@
       "types": "./dist/index.d.ts"
     },
     "./node": {
-      "import": "./dist/index.node.mjs",
-      "require": "./dist/index.node.cjs",
-      "default": "./dist/index.node.cjs",
-      "node": "./dist/index.node.cjs",
-      "types": "./dist/index.node.d.ts"
+      "import": "./dist/node/index.node.js",
+      "require": "./dist/node/index.node.js",
+      "default": "./dist/node/index.node.js",
+      "node": "./dist/node/index.node.js",
+      "types": "./dist/node/index.node.d.ts"
     }
+  },
+  "dependencies": {
+    "core-js": "^3.27.2"
```

Comment:
this would be needed because individual node files would pick up from this dependency what rollup is copying inside the bundle as strings.
If you look at the transpiled node code, some imports to core-js gets added